arch/lkl/kernel/time.c

```diff
@@ -121,7 +119,7 @@ void __init time_init(void)
 	if (ret)
 		pr_err("lkl: unable to register clocksource\n");
 
-	clockevents_config_and_register(&clockevent, HZ, 0, 0xffffffff);
+	clockevents_config_and_register(&clockevent, NSEC_PER_SEC, 1, ULONG_MAX);
```

Comment:
Among all changes in this PR, changing min_delta from 0 to 1 is actually where it 50% performance boost comes from. Is 0 min_delta a misconfiguration? I'm curious the reason behind this.